test/config/ytt/overlay-test-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+#@ load("@ytt:data", "data")
```

Comment:
I'd call `overlay-test-namespace.yaml` - `remove-test-namespaces.yaml` to be clear what's happening with the overlay